src/definitions/modules/modal.js

```diff
@@ -263,6 +267,13 @@
 
                 observeChanges: function () {
                     if ('MutationObserver' in window) {
+                        if ('ResizeObserver' in window && settings.observeChanges) {
```

Comment:
Does modal use the standard `MutationObserver` for anything else than size? If no, `ResizeObserver` should be enough when supported by the browser.

Reply:
The mutationobserver is still needed for the autofocus and tabbing functionality